core/src/main/java/org/akaza/openclinica/bean/extract/DownloadDiscrepancyNote.java

```diff
@@ -514,6 +677,62 @@ public void downLoadDiscBeans(List<DiscrepancyNoteBean> listOfBeans,
 
     }
 
+    public void downLoadThreadedDiscBeans(List<DiscrepancyNoteThread> listOfThreadedBeans,
```

Comment:
Instead of creating a new method like this, update the existing method to take delimiter as argument and the value for it would be null for PDF formats.

Reply:
done -- please see changes in new PR:
https://github.com/OpenClinica/OpenClinica/pull/3463